pkg/registry/core/namespace/storage/storage_test.go

```diff
@@ -189,6 +191,143 @@ func TestDeleteNamespaceWithCompleteFinalizers(t *testing.T) {
 	}
 }
 
+func TestDeleteWithGCFinalizers(t *testing.T) {
+	storage, server := newStorage(t)
+	defer server.Terminate(t)
+	defer storage.store.DestroyFunc()
+
+	propagationBackground := metav1.DeletePropagationBackground
+	propagationForeground := metav1.DeletePropagationForeground
+	propagationOrphan := metav1.DeletePropagationOrphan
```

Comment:
nit: these definitions don't seem to improve readability.

Reply:
`metav1.DeletePropagationBackground`(and the other twos) are constant, as a result, I can not take their address via `&metav1.DeletePropagationBackground`, I have to declare a new variable first.

Reply:
Gotcha.